kore/src/Kore/Step/Rule.hs

```diff
@@ -5,13 +5,18 @@ License     : NCSA
 
 -}
 
+{-# LANGUAGE TemplateHaskell #-}
```

Comment:
We are adding this to more and more files. Is it true that it increases compilation time by a lot?

Reply:
Yes, significantly. It's not necessary for using lenses, I'm just being lazy. I will think about how we should handle this in general.